src/gui/sdlmain.cpp

```diff
@@ -823,6 +823,28 @@ static const VsyncPreference &get_vsync_preference()
 	                              : sdl.vsync.when_windowed;
 }
 
+static void set_vfr_backfill_tempo_from_rate(const double dos_rate_hz)
```

Comment:
The word "tempo" is not an established term in this context and it can create confusion. I struggled to make sense of it until I read this function: because you're remapping `*_rate_hz` values it must be a frequency too.

So why don't we call it what it is then: `vfr_backfill_rate_hz`? Then every ambiguity would be removed.

Reply:
Now dropped the `tempo` term completely and went with the more direct `dupe_countdown` (and `vfr_dupe_countdown` outside the scope of the VFR function).